scripts/pso/pso_param_generation.py

```diff
@@ -118,52 +96,64 @@ def initial_population_generator(random, args):
 
     for i in range(0, len(param_name)):
         if param_name[i] == PARAM_MAPREDUCE_TASK_IO_SORT_FACTOR:
-            sort_factor_index = i
+            mr_sort_factor_index = i
         elif param_name[i] == PARAM_MAPREDUCE_TASK_IO_SORT_MB:
-            sort_memory_index = i
+            mr_sort_memory_index = i
         elif param_name[i] == PARAM_MAPREDUCE_MAP_SORT_SPILL_PERCENT:
-            spill_percent_index = i
+            mr_spill_percent_index = i
         elif param_name[i] == PARAM_MAPREDUCE_MAP_MEMORY_MB:
-            map_memory_index = i
+            mr_map_memory_index = i
         elif param_name[i] == PARAM_MAPREDUCE_REDUCE_MEMORY_MB:
-            reduce_memory_index = i
+            mr_reduce_memory_index = i
         elif param_name[i] == PARAM_PIG_MAX_COMBINED_SPLIT_SIZE:
-            max_combined_split_size_index = i
+            pig_max_combined_split_size_index = i
+        elif param_name[i] == PARAM_SPARK_EXECUTOR_MEMORY:
```

Comment:
As you mentioned offline, it would make sense to send this together with the Spark changes.

Reply:
Done